src/lib/components/sections/footer.svelte

```diff
@@ -10,10 +11,14 @@
 				acm<span class="brand-em brand-light">CSUF</span>
 			</span>
 		</p>
-
-		<a href="/github" class="github" target="_blank" rel="noopener norefferer">
-			<GitHubIcon />
-		</a>
+		<div class="footer-link">
+			<a href="/bug" class="bug" target="_blank" rel="noopener norefferer">
```

Comment:
Nitpicking here, but can we use `/bugs` instead of `/bug`?

Reply:
done!